### ISSUE TITLE
Bug in AUROC/AUPRC calculation

### ISSUE THREAD

**Issue by 2533245542** [MLforHealth/MIMIC_Extract#69]
https://github.com/MLforHealth/MIMIC_Extract/blob/0e835016ba811279d6b32bf9da1e397eb1e34128/notebooks/mmd_grud_utils.py#L436

The above should be named `logits`, not `probabilities`. The model outputs logits instead of probabilities as defined below.

https://github.com/MLforHealth/MIMIC_Extract/blob/0e835016ba811279d6b32bf9da1e397eb1e34128/notebooks/mmd_grud_utils.py#L193

So in below, instead of getting the second column of the logits, we should apply softmax to convert it to probabilities.
https://github.com/MLforHealth/MIMIC_Extract/blob/0e835016ba811279d6b32bf9da1e397eb1e34128/notebooks/Baselines%20for%20Mortality%20and%20LOS%20prediction%20-%20GRU-D.ipynb?short_path=3b93c09#L439

Then it will become the correct input (probabilities) to the AUROC/AUPRC function.

https://github.com/MLforHealth/MIMIC_Extract/blob/0e835016ba811279d6b32bf9da1e397eb1e34128/notebooks/Baselines%20for%20Mortality%20and%20LOS%20prediction%20-%20GRU-D.ipynb?short_path=3b93c09#L443

**Comment by mmcdermott** [comment]
Thank you for raising this issue! You're right that we should probably rename `probs` to `logits`. However, [roc_auc_score](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_auc_score.html) in sklearn actually accepts logits as well as probabilities, so the output scores should be valid.
